slf4j/src/main/scala/org/typelevel/log4cats/slf4j/Slf4jLogger.scala

```diff
@@ -46,4 +49,7 @@ object Slf4jLogger extends Slf4jLoggerCompat {
 
   def fromSlf4j[F[_]: Sync](logger: JLogger): F[SelfAwareStructuredLogger[F]] =
     Sync[F].delay(getLoggerFromSlf4j[F](logger))
+
+  def fromBlockingSlf4j[F[_]: Sync](logger: JLogger): F[SelfAwareStructuredLogger[F]] =
+    Sync[F].blocking(getLoggerFromBlockingSlf4j[F](logger))
```

Comment:
Sorry 😂 can this one be `delay` as well? `getLoggerFromBlockingSlf4j` is just calling `new` right? To make an instance of our own class. Making a new instance of a class should not be blocking right?

Reply:
Done xd